clients/src/test/java/org/apache/kafka/clients/consumer/internals/CommitRequestManagerTest.java

```diff
@@ -367,6 +372,42 @@ public void testAutocommitEnsureOnlyOneInflightRequest() {
         assertPoll(1, commitRequestManger);
     }
 
+    @Test
```

Comment:
similar to the comment above - we autocommit on close - should the interceptor be triggered?

Reply:
Yes, they should. This is tested in the integration test `PlaintextConsumer.testAutoCommitIntercept` - which works in terms of interceptors, but I have to keep disabled in this PR because of KAFKA-16155. This PR does call the interceptors after closing the network thread (I pinged you about it above). I can add a little unit test to `AsyncKafkaConsumerTest`. I don't think we can add a unit test for it in `CommitRequestManagerTest`, because the autocommit on close is triggered from the application thread, and the interceptors are run from the application thread.